.github/workflows/manual-trigger-individual-test.yml

```diff
@@ -0,0 +1,9 @@
+# This is a basic workflow that is manually triggered
```

Comment:
q: why do we need a manual trigger?

Reply:
this will help on debugging tests that runs in pipeline vs locally